_sass/modules/_header.scss

```diff
@@ -1,65 +1,149 @@
-// header, the first grid row
-// it is also a grid for the logo and nav
-header {
-  grid-column: full;
-  display: grid;
-  padding: 2rem;
+$header-font-size-desktop: 1.2rem;
+
+.header {
+  font-family: $bold-font;
+  font-size: $base-font-size2;
+  padding: 2.2rem 0 1rem;
+
+  &.header--nav-open {
+    height: 100%;
+    position: fixed;
+    width: 100%;
+    z-index: 1;
+  }
+
   @media screen and (min-width: $small-desktop-min) {
-    grid-template-columns: 1fr auto;
-    grid-column-gap: 1rem;
+    @include flexbox;
+    @include justify-content(space-between);
+    font-size: $header-font-size-desktop;
+    height: auto;
+    position: static;
+  }
+}
+
+.header__brand {
+  @media screen and (max-width: $small-desktop-max) {
+    @include flexbox;
+    @include justify-content(space-between);
+    @include align-items(center);
+    padding-right: $u-space-small;
   }
 }
 
-// logo, is a grid item
-header > h1 {
-  display: inline-block;
-  padding-bottom: 1rem;
+.header__logo {
+  padding-left: $u-space-small;
+  margin-right: $u-space-small;
+  min-width: 11rem;
+
+  @media screen and (min-width: $desktop-min) {
+    height: 4rem;
+  }
+
+  @media screen and (min-width: $desktop-min) {
+    margin-right: 0;
+    padding-left: $u-space-default;
+    width: 15vw;
+  }
+
+  @media screen and (min-width: $large-desktop-min) {
+    width: 21vw;
+  }
+}
+
+.icon-menu-open {
```

Comment:
would use `menu` or `nav` consistently (got `icon-nav-close`) below. I'd also ocnsider using "expanded" instead of "open" to avoid the ambiguity (open as in action and open as in state)